nntrainer/src/activation_layer.cpp

```diff
@@ -51,20 +51,23 @@ int ActivationLayer::initialize(bool last) {
   return ML_ERROR_NONE;
 }
 
-Tensor ActivationLayer::forwarding(Tensor in, int &status) {
-  status = ML_ERROR_NONE;
+sharedTensor ActivationLayer::forwarding(sharedTensor in) {
+  input = in.get()[0];
+  hidden = _act_fn(input);
 
-  input = in;
-  hidden = _act_fn(in);
-
-  return hidden;
+  return MAKE_SHARED_TENSOR(hidden);
```

Comment:
```suggestion
  return MAKE_SHARED_TENSOR(std::move(hidden));
```

Reply:
Can we do this? hidden is the class object variable and not local.

Reply:
Oh I missed that. nvm